test/test_flake8.py

```diff
@@ -11,13 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from flake8.api.legacy import get_style_guide
```

Comment:
The empty lines were shuffled here. There should be one empty line between the copyright and the first import and no empty lines between the imports (since they are both in the same group).